### PR TITLE
fix return type of getPuppeteer in typescript definitions

### PR DESCRIPTION
**Pull request by SrinivasanTarget**
## Proposed changes

fix return type of getPuppeteer in typescript definitions

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [ ] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

**Comment by christian-bromann** [comment]
@SrinivasanTarget thanks for the PR. I think we can't get around this failing dependency test which is why I would suggest to ignore this case in [the test](https://github.com/webdriverio/webdriverio/blob/master/scripts/depcheck.js#L35) by adding a new filter.

**Comment by SrinivasanTarget** [comment]
@christian-bromann sure, I will look into excluding it.

File: scripts/depcheck.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ const EXEC_OPTIONS = { silent: true, async: true }
         result.package = pkg
         result.packagePath = packagePath
         return result
-    }))).filter((result) => Object.keys(result.missing).length)
+    }))).filter((result) => Object.keys(result.missing).length && Object.keys(result.missing) != 'puppeteer')
```
Comment (SrinivasanTarget) [inline scripts/depcheck.js:35]: @christian-bromann Added a filter. Do you have any better ways to filter it?

Comment (christian-bromann) [reply scripts/depcheck.js:35]: Let's define a constant that defines all deficits we accept and match them against `package` and `packagePath`.

**Comment by osmolyar** [comment]
@SrinivasanTarget , @christian-bromann , when trying to update my fork to master it gave this error at "git push -f origin master" after rebasing:
`C:\webdriverio\packages\devtools\src\commands\performActions.js
  10:32  error  Unable to resolve path to module 'puppeteer-core/lib/cjs/common/USKeyboardLayout'  import/no-unresolved

✖ 1 problem (1 error, 0 warnings)
`
(It works with --no-verify).  Not sure if that is the test you're talking about skipping above

**Comment by christian-bromann** [comment]
@osmolyar please run `npm run setup-full` to download all package dependencies as we have updated Puppeteer across all packages.

**Pull request merged**